cumulusci/salesforce_api/package_install.py

```diff
@@ -48,6 +48,43 @@ class PackageInstallOptions(CCIModel):
     password: Optional[str]
     security_type: SecurityType = SecurityType.FULL
 
```

Comment:
Should activate_remote_site_settings really be optional? We don't want it to be possible to explicitly set it to null, do we?

Reply:
No, we don't - correcting this.